o2a/mappers/subworkflow_mapper.py

```diff
@@ -99,26 +105,26 @@ def _parse_config(self):
                 for node in property_nodes:
                     name = node.find("name").text
                     value = el_utils.replace_el_with_var(
-                        node.find("value").text, params=self.params, quote=False
+                        node.find("value").text, self.property_set, quote=False
                     )
-                    self.properties[name] = value
+                    self.job_properties[name] = value
 
-    def to_tasks_and_relations(self):
-        tasks = [
+    def to_tasks_and_relations(self) -> Tuple[List[Task], List[Relation]]:
```

Comment:
The parent class has type definitions. Is this change necessary?

Reply:
I think it's much nicer to add it here for potential reader in the future. This way you do not have to look at the base class to see what types should be returned. And mypy will complain if those will differ. So I'd rather leave it here.

Please resolve it @mik-laj  if you are ok with it.